stdlib/Pkg.version

```diff
@@ -1,4 +1,4 @@
 PKG_BRANCH = master
```

Comment:
I think the branch should be release-1.8?

I'm not sure it matters though?

Reply:
I think it matters when you do `DEPS_GIT`. I'll update it.